http-client/src/main/java/io/airlift/http/client/HttpClientBinder.java

```diff
@@ -46,7 +49,12 @@ public HttpClientBindingBuilder bindHttpClient(String name, Class<? extends Anno
     {
         HttpClientModule module = new HttpClientModule(name, annotation);
         binder.install(module);
-        return new HttpClientBindingBuilder(module, newSetBinder(binder, HttpRequestFilter.class, annotation));
+        return new HttpClientBindingBuilder(module, newSetBinder(binder, HttpRequestFilter.class, annotation), newSetBinder(binder, HttpStatusListener.class, annotation));
+    }
+
+    public LinkedBindingBuilder<HttpStatusListener> addGlobalStatusListenerBinding()
```

Comment:
Nit: move this below `addGlobalFilterBinding()` so that the ordering matches the field ordering